README.md

```diff
@@ -92,7 +92,8 @@ event := models.EventModel{
 	Response:     rsp,
 	SessionToken: &sessionToken,
 	Tags:         nil,
-	UserId:       &userId,
+  UserId:       &userId,
```

Comment:
are these aligned with lines above ?

Reply:
Updated the PR which fixes the indentation issue as Markdown document and preview shows different results in VSCode.